sensors/reed/README.md

```diff
@@ -0,0 +1,21 @@
+# Reed sensor example
+
+This example prints 'contact' and 'no contact' depending on the state of the reed switch.
+
+## Running the example
+
+Wire your ESP32 and the reed sensor according to:
+
+![Alt text](./reed_breadboard.png "Reed example - breadboard overview")
+
+![Alt text](./reed_schematic.png "Reed example - schematic circuit")
+
+To run the example on your ESP32, execute the following command:
+
+```bash
+toit run reed.toit
+```
+
+## License
+
+[BSD0](https://choosealicense.com/licenses/0bsd/)
```

Comment:
Alternative would be to point to the LICENSE file that is in the root of the repository.